config/nginx.conf.erb

```diff
@@ -27,14 +40,17 @@ http {
 
     server {
         listen <%= ENV["PORT"] %>;
+        server_name micromasters.herokuapp.com;
+        rewrite https://micromasters.mit.edu$request_uri permanent;
+    }
+
+    server {
+        listen <%= ENV["PORT"] %>;
         server_name _;
         root /app;
 
-        if ($host = 'micromasters.herokuapp.com') {
-            return 301 'https://micromasters.mit.edu';
-        }
-
         location ^~ /static/(.*$) {
+            expires max;
```

Comment:
That will set the expires and cache-control headers for static assets served through Nginx.